runtime/vm/vmprops.c

```diff
@@ -1413,7 +1413,7 @@ getMUtf8String(J9JavaVM *vm, const char *userString, UDATA stringLength)
 		}
 		result = mutf8Buffer;
 	} else {
-#if defined(WIN32)
+#if defined(WIN32) || defined(OSX)
```

Comment:
Suggest tests should be sorted alphabetically:
```c
#if defined(OSX) || defined(WIN32)
```
and the `#else` and `#endif` comments should be updated so they still match.